src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -90,6 +91,11 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
 
         self.__current_operation = None
 
+        self.__truncation_timestamp_tracker = None  # To allow truncation on upcoming operation when tracker > 60 sec (tentatively)
```

Comment:
We had discussed this to be self.__truncation_timestamp yesterday

Reply:
removed tracker, and other reference
done